src/functTest/groovy/com/bmuschko/gradle/docker/tasks/DockerMultipleClientFunctionalTest.groovy

```diff
@@ -16,15 +16,15 @@ class DockerMultipleClientFunctionalTest extends AbstractGroovyDslFunctionalTest
                 }
             }
             task dockerClient2(type: DockerOperation) {
-                url = 'tcp://docker.corp.com'
+                url = 'tcp://docker.corp.com:2375'
```

Comment:
Why was this change required here?

Reply:
Unlike the JerseyDockerHttpClient 
`case "tcp":
                try {
                    dockerHost = new URI(dockerHost.toString().replaceFirst("tcp", protocol));`
 
the OkDockerHttpClient 
`           case "tcp":
                baseUrlBuilder = new HttpUrl.Builder()
                    .scheme(isSSL ? "https" : "http")
                    .host(dockerHost.getHost())
                    .port(dockerHost.getPort());
                break;`
has a specific call to getPort. This returns -1 where no port is set, and you get a failure trying to connect to port -1
|      * What went wrong:
|      Execution failed for task ':dockerInfo'.
|      > unexpected port: -1

I could alter the URI to set a default port, or detect this and output a more accurate message: -
`			if (config.getDockerHost().getScheme() == "tcp" && config.getDockerHost().getPort() == -1) {
				throw new GradleException("Dockerhost uri with tcp scheme must contain port")
			}`
and add a test, but neither seemed to add much.

Reply:
The change is correct. The port should be specified explicitly, there is no guarantee that `2375` will be implicitly used, and it is there in Jersey only for historical reasons

Reply:
Got it, thanks.